src/stream-tcp.c

```diff
@@ -4841,18 +4841,13 @@ int StreamTcpPacket (ThreadVars *tv, Packet *p, StreamTcpThread *stt,
         /* check for conditions that may make us not want to log this packet */
 
         /* streams that hit depth */
-        if ((ssn->client.flags & STREAMTCP_STREAM_FLAG_DEPTH_REACHED) &&
+        if ((ssn->client.flags & STREAMTCP_STREAM_FLAG_DEPTH_REACHED) ||
```

Comment:
I believe I've seen this before. This would lead to potential detection and logging bypass. E.g. depth 1MiB, http post > 1MiB, response is not logged even though it's below the depth for its direction.

Reply:
Yes, it was sent once but your explanation looks better this time :)